Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -180,15 +182,15 @@ Acts::Result<double> Acts::EigenStepper<E, A>::step(
     }
 
     // Compute and check the local integration error estimate
-    error_estimate = std::max(
+    error_estimate =
         h2 * ((sd.k1 - sd.k2 - sd.k3 + sd.k4).template lpNorm<1>() +
-              std::abs(sd.kQoP[0] - sd.kQoP[1] - sd.kQoP[2] + sd.kQoP[3])),
-        1e-20);
+              std::abs(sd.kQoP[0] - sd.kQoP[1] - sd.kQoP[2] + sd.kQoP[3]));
+    error_estimate = std::max(error_estimate, 1e-20);
 
     return success(error_estimate <= state.options.tolerance);
   };
 
-  double stepSizeScaling = 1.;
+  double stepSizeScaling = 1;
```

Comment:
This doesn't make a numerical difference, right?

Reply:
should not but I will revert it